lib/can_do.rb

```diff
@@ -26,8 +28,8 @@
 #
 class CanDo
   include Singleton
-
-  NotAFeature = Class.new(StandardError)
+  FEATURE_KEY_PREFIX = "features/"
```

Comment:
I think the traditional separator in the Redis world is `:` (colon) - not `/`. Sidekiq does that and I've also found [similar stories](https://stackoverflow.com/questions/6965451/redis-key-naming-conventions).